Automattic/wp-calypso
# Stats: Updating Emails component on the Subscribers page

Related to https://github.com/Automattic/red-team/issues/99

## Proposed Changes

* replaces old empty state on the Subscribers page
* removed old component wrapping `StatsModule`
* removing feature flag check from that component since it has been released

## Why are these changes being made?


* unifying modules and states

## Testing Instructions



* open the live branch
* navigate to a new site that has at least two subscribers but no emails sent
* verify that the new empty state shows up
* verify that the component works properly for a page with data

## Pre-merge Checklist
